src/containers/User/UserArea.tsx

```diff
@@ -305,11 +330,25 @@ const MainMenuBar: React.FC<MainMenuBarProps> = ({
             </Dropdown>
           </List.Item>
         )}
+        {isMobile && <List.Item onClick={() => logout()}>{t('MENU_LOGOUT')}</List.Item>}
```

Comment:
Couple of things:
- I think we should put the other items from the "User" menu in here too, namely "Edit User Details" and "Change Language" 
- Could we put a "spacer" (e.g.`<hr/>`) between the "normal" menu and the extra "Logout" etc?
- Accidentally logging out is kind of annoying. Can we put a confirmation on this? Just use the existing "useConfirmationModal", and search for it to see how it's used.

Reply:
Yep, will get onto this.

Reply:
@CarlosNZ - There is actually a bit more to this than I thought.  I might have to break out some of the code in UserMenu such as the Modal etc.. and there is some filtering of the templates that I am not entirely familiar with yet...  I'll give it a go and let you know.

Reply:
Oh, I thought you should be able to just copy the items from the UserMenu into here. You can leave it for now if it's too fiddly and we can deal with it in a separate issue?